laborator/content/operatii-memorie-gdb/4-find-max/find-max.c

```diff
@@ -41,7 +41,7 @@ int compare(const void *a, const void *b);
  * Se citeste de la tastatura un vector si se cere sa se afle
  * elementul maxim folosind functia find_max.
  * Rezultatul functiei find_max trebuie cast la int, spre exemplu:
- * int *res = (*int) find_max(...);
+ * int *res = (int *) find_max(...);
```

Comment:
In fact, this suggestion is wrong altogether. `void *` is automatically cast to the type of the left-hand operator in an assignment. So I suggest you simply remove these lines.